draft-fossati-core-cf-reg-update.md

```diff
@@ -114,46 +117,53 @@ Other than that, it does not change the Security Considerations of {{-coap}}.
 
 # IANA Considerations {#iana}
 
-## Expert Review with Expert Check "FCFS+"
-
-This document introduces the term "Expert Review (Expert Check: FCFS+)" to describe a registration policy that would typically have been handled as FCFS, except the approval checklist is complex enough to require advice from a DE.
-Such policy can be viewed as a "lightweight" form of the "full" Expert Review.
-
-A protocol that requires "Expert Check: FCFS+" for registration of its parameters MUST specify what the FCFS+ checks entail.
+The CoAP Content-Formats registration procedures defined in {{Section 12.3 of -coap}} are modified as shown in {{tbl-new-cf-proc}}.
 
-## CoAP Content-Formats Registration Procedures Update
-
-The CoAP Content-Formats registration procedures defined in {{Section 12.3 of -coap}} are modified to transition the 10000-64999 range from FCFS to "Expert Review (Expert Check: FCFS+)", with the FCFS+ checklist described in {{checks}}.
-
-| Range | Registration Procedures |
+| Range | Registration Procedures | Note |
 |--------|--------|
-| 0-255 | Expert Review (Full) |
-| 256-9999 | IETF Review or IESG Approval |
-| 10000-64999 | Expert Review (Expert Check: FCFS+) |
+| 0-255 | Expert Review (Full) | |
```

Comment:
Here it would be better to place the "(Full)" into the Note column. So something like:
| Expert Review | Full review

or 
| Expert Review | Full review described in [RFCthis], Section X

in case we have any description of what "full" review means. For example we have the sentence "the DE must also evaluate the requested codepoint concerning the limited availability of the 1-byte codepoint space." which is already describing a part of the "full" review.

Reply:
This is a very good point.  We have left "full" to the discretion of the DE, but it seems like a good idea to go the full mile.

Reply:
@EskoDijk, please see if [0369f8e](https://github.com/thomas-fossati/draft-cf-reg-update/pull/17/commits/0369f8e1d93b08290cfd4e5bf55486d9f3f21106) works for you.

Reply:
If needed, we can add more extra checks for "full" reviewers over "lightweight" ones.